protocol/testing/e2e/gov/add_new_market_test.go

```diff
@@ -302,7 +302,7 @@ func TestAddNewMarketProposal(t *testing.T) {
 				require.Equal(t, initPerpetuals, tApp.App.PerpetualsKeeper.GetAllPerpetuals(ctx))
 				require.Equal(t, initClobPairs, tApp.App.ClobKeeper.GetAllClobPairs(ctx))
 			case govtypesv1.ProposalStatus_PROPOSAL_STATUS_PASSED:
-				// Proposal passed and succesfully executed, check states are updated.
```

Comment:
Let me know if I'm missing something, but were we able to add a regression test for the issue we saw?

Specifically, if an order removal occurs in `CheckTx`, then it no longer decrements the stateful order count because it's not committed to the `deliverState`?

Reply:
Not yet, was prioritizing the fix over the test.